build/diva/install.sh

```diff
@@ -0,0 +1,14 @@
+echo "---- INSTALL Diva  ----"
+
+divabranch=master
+
+# cleanup if it exists
+rm -rf diva
+
+# get diva
+git clone --branch v6.0.2 git://github.com/DDMAL/diva.js.git divajs || { echo "Diva git clone failed" ;exit 1; }
+cd divajs
+git checkout $divabranch
```

Comment:
Since we're doing the `--branch v6.0.2` flag in the clone I don't think you need line 3 and 11. Do a quick check to see if you can remove them, then 👍.